packages/infra/bin/infrastructure.ts

```diff
@@ -50,11 +51,22 @@ async function deploy(config: EnvConfig) {
   // 4. Deploy the HL7 Notification Routing stack.
   //---------------------------------------------------------------------------------
   if (!isSandbox(config)) {
-    new Hl7NotificationStack(app, "Hl7NotificationStack", {
+    const hl7NotificationStack = new Hl7NotificationStack(app, "Hl7NotificationStack", {
       env,
       config,
       version,
     });
+
+    config.hl7Notification.vpnConfigs.forEach((config, index) => {
+      const vpnStack = new VpnStack(app, `VpnStack-${config.partnerName}`, {
+        vpnConfig: { ...config },
+        index,
+        networkStackId: "NestedNetworkStack",
```

Comment:
Can't we get this form the `hl7NotificationStack.networkStack`?

Reply:
This works fine:
```
networkStackId: "NestedNetworkStack"
```

This gives me an error. I think its because I'm passing these values around outside the context of a stack, in a script.
```
networkStackId: hl7NotificationStack.networkStack.stackId,
...
> Error: Stack "VpnStack-MyTestHIE" cannot reference {Hl7NotificationStack/NestedNetworkStack.NestedStack/NestedNetworkStack.NestedStackResource[Ref]} in stack "Hl7NotificationStack". Cross stack references are only supported for stacks deployed to the same account or between nested stacks and their parent stack
```

I could make a constant and use it in both places?